docs/APIReference-SelectionState.md

```diff
@@ -47,6 +47,10 @@ For instance, when extracting a slice of text from a block based on a
 `SelectionState`, it is irrelevant whether the selection is backward:
 
 ```
+var selection = editorState.getSelection();
```

Comment:
@flarnie I have made appropriative changes: change the name for the selection variable, and use currentContentBlock in spite of the myContentBlock, as for me also it makes more sense